release/mk-release-snapshots.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#
+# Publish Alpha releases (a.k.a. daily snapshot)
+#
+
+. ~/.server.config || exit $?
+
+# auto-detect the latest 'major' version number
+SQUID_RELEASE=`ls -1 $SQUID_VCS_PATH | cut -d- -f2 | sort -h | tail -n 1`
+
+# file Jenkins uses to assemble the list of snapshot files
+outfile=HEAD
```

Comment:
Based on this variable usage, it is not the name of the described file, but its "base" name. I suggest reducing code duplication by moving the extension from the code below to here. The suggestion below shows only the filename part of that change:

```suggestion
outfile=HEAD.out
```

Reply:
Sounds sensible; I support the change

Reply:
Done in [fd084e3](https://github.com/squid-cache/ci/pull/13/commits/fd084e3e1f935bab8377ddc98d7766f6048db9af)